code/network/psnet2.h

```diff
@@ -32,6 +32,7 @@
 // kazan - I think this should raise the ships limit across the network
 //#define MAX_PACKET_SIZE 4096
 #define MAX_PACKET_SIZE		512
+#define MAX_PACKET_DESC_LEN MAX_PACKET_SIZE - 12
```

Comment:
Just move this to `multimsgs.cpp` since that's the only place it's used

Reply:
This one was actually referenced in a few places, so it seemed to be best to move this into the strings section in multi.h.